git-features/Cargo.toml

```diff
@@ -20,14 +20,17 @@ progress = ["prodash"]
 
 ## If set, and with the `parallel` feature set, walkdir iterators will be single-threaded.
 ## This feature exists to avoid [certain side-effects](https://github.com/starship/starship/issues/4251) of rayon threadpool configuration with `jwalk`.
-fs-walkdir-single-threaded = []
+fs-walkdir-parallel = ["parallel", "jwalk" ]
```

Comment:
This is definitely an improvement and cleaner, as single-threaded directory walking is best done by `walkdir` without the added dependencies.